wallet/src/de/schildbach/wallet/ui/coinjoin/CoinJoinLevelFragment.kt

```diff
@@ -53,11 +54,11 @@ class CoinJoinLevelFragment : Fragment(R.layout.fragment_coinjoin_level) {
             lifecycleScope.launch {
                 if (viewModel.isMixing) {
                     if (confirmStopMixing()) {
-                        viewModel.stopMixing()
+                        viewModel.setMode(CoinJoinMode.NONE)
```

Comment:
Setting the `CoinJoinMode` will trigger the `CoinJoinService` to start or stop mixing.